client/src/amplify-ui/cli-feature-flag/feature-flags.json

```diff
@@ -314,6 +314,27 @@
             "defaultExistingProject": true
           }
         ]
+      },
+      "handleListNullabilityTransparently": {
+        "description": "Configure the nullability of the List types in Datastore Models generation",
+        "type": "Feature",
+        "valueType": "Boolean",
+        "versionAdded": "4.50.2",
```

Comment:
This should be up-to-date with the latest version of CLI when releasing the feature